static/styles/lib/alerts.scss

```diff
@@ -11,18 +11,26 @@
 	.dropdown-menu {
 		width: 400px;
 	}
-
-	@media (max-width: 450px) {
-		.dropdown-menu {
-			width: 70vw;
-			margin-left: 80px;
+	@media (min-width: 381px) and (max-width: 420px) {
+		.minimal-button .dropdown-menu {
+			left: -18.2rem!important;
```

Comment:
1. same here with negative values
2. might be a bit petty, but please put a blank between the unit and the "!important"